ingestion_tools/scripts/data_validation/conftest.py

```diff
@@ -53,15 +49,10 @@ def get_voxel_spacing_files(bucket: str, dataset: str, run_names: List[str], vox
         for result in results:
             tentatives += result
 
-    exclude = ["Images", "dataset_metadata.json"]
-
-    for ex in exclude:
-        tentatives = [tent for tent in tentatives if ex not in tent]
-
     return tentatives
 
 
-def voxel_spacings(voxel_spacing_files: List[str]) -> List[float]:
+def voxel_spacings(voxel_spacing_files: List[str]) -> List[str]:
```

Comment:
voxel spacings are going to be kept as strings, because we want to preserve the folder name (and ensure it's exactly 3 digits of precision)